test/acceptance/view_benchmark_graphs_test.rb

```diff
@@ -72,9 +72,60 @@ class ViewBenchmarkGraphsTest < AcceptanceTest
     assert_not page.has_content?("#{benchmark_run_category_humanize} memory Graph")
   end
 
+  test "User should be able to view long running benchmark graphs when they go back and forward 
+    in the broswer window" do
+
+    javascript_driver = Capybara.javascript_driver
+    default_driver = Capybara.current_driver
+    Capybara.javascript_driver = :selenium
+    Capybara.current_driver = :selenium
```

Comment:
I don't think we need selenium for this test. One of the other tests was using selenium because I couldn't get `capybara-webkit` to click on the elements in the graph.